bolt-modules/boltlib/lib/puppet/functions/set_var.rb

```diff
@@ -0,0 +1,29 @@
+require 'bolt/error'
+
+# Sets a variable { key => value } for a target
+#
+# This function takes 3 parameters:
+# * A Target object to set the variable for
+# * The key for the variable (String)
+# * The value of the variable (Data)
+#
+# Returns Undef
```

Comment:
If you want to be precise say "Returns Undef value", or "Returns undef" (since `Undef` is the data type it is possible to misunderstand).
Also `.`